blog/_src/posts/2017-05-24-plc-russia.md

```diff
@@ -0,0 +1,39 @@
+    Title: Programming Language Conference in Russia
+    Date: 2017-05-24T12:25:17
+    Tags: by Artem Pelenitsyn
+
```

Comment:
I think we need to add a line here that says:

```
<!-- more -->
```

otherwise the entire post will appear as its abstract. I think.